macros/largefile-check.m4

```diff
@@ -20,11 +20,8 @@ define(WX_SYS_LARGEFILE_MACRO_VALUE,
 [
     AC_CACHE_CHECK([for $1 value needed for large files], [$3],
         [
-          AC_TRY_COMPILE([#define $1 $2
-                          #include <sys/types.h>],
-                         WX_SYS_LARGEFILE_TEST,
-                         [$3=$2],
-                         [$3=no])
+          AC_COMPILE_IFELSE([AC_LANG_PROGRAM([[#define $1 $2
+                          #include <sys/types.h>]], [[WX_SYS_LARGEFILE_TEST]])],[$3=$2],[$3=no])
```

Comment:
This makes the largefile check functional:

```
          AC_COMPILE_IFELSE([AC_LANG_PROGRAM([[#define $1 $2
                          #include <sys/types.h>], WX_SYS_LARGEFILE_TEST])],[$3=$2],[$3=no])
```

Reply:
Fix applied!